app/models/taxon.rb

```diff
@@ -50,7 +52,11 @@ def base_path=(base_path)
   end
 
   def base_path
-    @base_path ||= '/' + [path_prefix, path_slug].compact.join('/')
+    if [path_prefix, path_slug].any?
+      '/' + [path_prefix, path_slug].compact.join('/')
+    else
+      @base_path
+    end
```

Comment:
This also fixes a UI problem that we've had for some time, e.g. for the following taxon that has a base path of `/imported-topic/topic/...`: https://content-tagger.integration.publishing.service.gov.uk/taxons/fd074d43-dca1-4bce-ab32-334e7a50d017

Development (this PR):
<img width="594" alt="screen shot 2018-03-01 at 12 46 26" src="https://user-images.githubusercontent.com/885223/36845453-b6fe267e-1d4e-11e8-9961-01b6aacf3255.png">

Integration (latest master):
<img width="593" alt="screen shot 2018-03-01 at 12 46 37" src="https://user-images.githubusercontent.com/885223/36845462-bdb27ba0-1d4e-11e8-9093-09ac13557661.png">